tests/PrestaShopBundle/Routing/Converter/LegacyUrlConverterTest.php

```diff
@@ -152,14 +153,12 @@ public function testMultipleLegacyLinks()
 
     /**
      * If a non existent action is used in the url (meaning one that has not been
-     * migrated yet) it must not return the index
-     * @throws ArgumentException
-     * @throws RouteNotFoundException
+     * migrated yet) it must not return the index route but throw an Exception instead
      */
     public function testNonExistentAction()
```

Comment:
Maybe it's easier to do it using expectException() helper? https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions.examples.ExceptionTest.php

It's ok to split your assertions in more tests.

Reply:
Yes I know, and you can use the `@expectedException` as well, but I was a bit lazy in looking for a method names for each tests ^^
This wouldn't have made the reading of the tests easier either
But you insist on changing it, maybe I can make something cleaner with a `@dataProvider`

Reply:
I don't insist, I'm already happy to see people write tests :+1: